staging/src/k8s.io/component-base/logs/datapol/externaltypes.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package datapol
+
+import (
+	"net/http"
+	"reflect"
+
+	"golang.org/x/oauth2"
```

Comment:
Those two imports introduce additional dependecies to all components using `component-base`. To make review faster I would propose move handling of those types to separate PR.